astropy/table/table.py

```diff
@@ -1903,6 +1903,52 @@ def remove_rows(self, row_specifier):
         if hasattr(self, '_groups'):
             del self._groups
 
+    def remove_na(self):
```

Comment:
Call this `remove_masked_rows`, and remove all references to N/A.  "Masking" is the concept used in astropy Table, not N/A.